### ISSUE TITLE
Just packaging app again causes crashing

### ISSUE THREAD

**Issue by Avamander** [iBotPeaches/Apktool#2112]
### Information
1. **Apktool Version)** - 2.4.0
2. **Operating System** - Linux
3. **APK From?** - PlayStore

### Stacktrace/Logcat

```
E AndroidRuntime: java.lang.RuntimeException: Unable to start activity ComponentInfo{com.hughesmedia.big_finish/com.hughesmedia.big_finish.LoginActivity_}: android.view.InflateException: Binary XML file line #14: Binary XML file line #14: Error inflating class EditText
E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2946)
E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3081)
E AndroidRuntime:        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1831)
E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
E AndroidRuntime:        at android.os.Looper.loop(Looper.java:201)
E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6815)
E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)
E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #14: Binary XML file line #14: Error inflating class EditText
E AndroidRuntime: Caused by: android.view.InflateException: Binary XML file line #14: Error inflating class EditText
E AndroidRuntime: Caused by: java.lang.StringIndexOutOfBoundsException: length=0; index=0
E AndroidRuntime:        at java.lang.String.charAt(Native Method)
E AndroidRuntime:        at com.android.internal.util.XmlUtils.convertValueToInt(XmlUtils.java:107)
E AndroidRuntime:        at android.content.res.TypedArray.getInt(TypedArray.java:385)
E AndroidRuntime:        at android.widget.TextView.<init>(TextView.java:1194)
E AndroidRuntime:        at android.widget.EditText.<init>(EditText.java:88)
E AndroidRuntime:        at android.widget.EditText.<init>(EditText.java:84)
E AndroidRuntime:        at android.support.v7.widget.AppCompatEditText.<init>(AppCompatEditText.java:69)
E AndroidRuntime:        at android.support.v7.widget.AppCompatEditText.<init>(AppCompatEditText.java:65)
E AndroidRuntime:        at android.support.v7.app.AppCompatViewInflater.createEditText(AppCompatViewInflater.java:192)
E AndroidRuntime:        at android.support.v7.app.AppCompatViewInflater.createView(AppCompatViewInflater.java:114)
E AndroidRuntime:        at android.support.v7.app.AppCompatDelegateImpl.createView(AppCompatDelegateImpl.java:1266)
E AndroidRuntime:        at android.support.v7.app.AppCompatDelegateImpl.onCreateView(AppCompatDelegateImpl.java:1316)
E AndroidRuntime:        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:776)
E AndroidRuntime:        at android.view.LayoutInflater.createViewFromTag(LayoutInflater.java:734)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:867)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:870)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:870)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:870)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:870)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.rInflate(LayoutInflater.java:870)
E AndroidRuntime:        at android.view.LayoutInflater.rInflateChildren(LayoutInflater.java:828)
E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:519)
E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:427)
E AndroidRuntime:        at android.view.LayoutInflater.inflate(LayoutInflater.java:374)
E AndroidRuntime:        at android.support.v7.app.AppCompatDelegateImpl.setContentView(AppCompatDelegateImpl.java:469)
E AndroidRuntime:        at android.support.v7.app.AppCompatActivity.setContentView(AppCompatActivity.java:140)
E AndroidRuntime:        at com.hughesmedia.big_finish.LoginActivity_.setContentView(LoginActivity_.java:53)
E AndroidRuntime:        at com.hughesmedia.big_finish.LoginActivity_.onCreate(LoginActivity_.java:39)
E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7210)
E AndroidRuntime:        at android.app.Activity.performCreate(Activity.java:7201)
E AndroidRuntime:        at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1272)
E AndroidRuntime:        at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2926)
E AndroidRuntime:        at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:3081)
E AndroidRuntime:        at android.app.servertransaction.LaunchActivityItem.execute(LaunchActivityItem.java:78)
E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.executeCallbacks(TransactionExecutor.java:108)
E AndroidRuntime:        at android.app.servertransaction.TransactionExecutor.execute(TransactionExecutor.java:68)
E AndroidRuntime:        at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1831)
E AndroidRuntime:        at android.os.Handler.dispatchMessage(Handler.java:106)
E AndroidRuntime:        at android.os.Looper.loop(Looper.java:201)
E AndroidRuntime:        at android.app.ActivityThread.main(ActivityThread.java:6815)
E AndroidRuntime:        at java.lang.reflect.Method.invoke(Native Method)
E AndroidRuntime:        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:547)
E AndroidRuntime:        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:873)

```

### Steps to Reproduce
1. `apktool d source.apk`
2. `apktool b source source.apk`
3. Install it, it crashes

### APK
https://play.google.com/store/apps/details?id=com.hughesmedia.big_finish

### Questions to ask before submission
1. Have you tried `apktool d`, `apktool b` without changing anything? Yep
2. If you are trying to install a modified apk, did you resign it? Yep
3. Are you using the latest apktool version? Yep

**Comment by jayce211** [comment]
Did you sign it again? I notice when I unpack and build if I don't sign the file it sometimes won't install and if it does it crashes.

**Comment by iBotPeaches** [comment]
This should have been fixed in v2.4.1. This was related to compressed resources so they couldn't be inflated properly. Sorry it took a year+ to respond. If you still encounter errors with the upcoming v2.5.0 release, come back to this ticket and I can take another triage look.

Closing.
